### DISCUSSION TITLE
GPU selection

### DISCUSSION THREAD

**Discussion by startergo** [HJebbour/HP-ProDesk-400-G1-Hackintosh-every-macOS#3] [Q&A] [answered]
Hey nice project! Congrats on the success! I don't really need Tiger, but I definitely want to use SL. What would be the best GPU for SL? I prefer Kepler and I have GTX Titan.

**Comment by HJebbour** [comment, marked as answer]
You can check [here](https://dortania.github.io/OpenCore-Install-Guide/macos-limits.html#gpu-support) if your GPU is supported on Snow Leopard.
